x-pack/plugins/cases/server/services/cases/transform.ts

```diff
@@ -20,6 +20,8 @@ import type { ESCaseAttributes, ExternalServicesWithoutConnectorId } from './typ
 import {
   CONNECTOR_ID_REFERENCE_NAME,
   PUSH_CONNECTOR_ID_REFERENCE_NAME,
+  SEVERITY_ESMODEL_TO_EXTERNAL,
```

Comment:
Could you please add tests in `x-pack/plugins/cases/server/services/cases/transform.test.ts` to test all changes regarding severity?

Reply:
Also here `x-pack/plugins/cases/server/services/cases/index.test.ts`